Lib/test/test_pty.py

```diff
@@ -38,10 +38,6 @@ def normalize_output(data):
     if data.endswith(b'\r\r\n'):
         return data.replace(b'\r\r\n', b'\n')
 
-    # IRIX apparently turns \n into \r\n.
-    if data.endswith(b'\r\n'):
-        return data.replace(b'\r\n', b'\n')
```

Comment:
It is just the comment that is IRIX-specific, not the code. I believe the code is necessary to make the tests pass e.g. on Linux, by undoing the ONLCR terminal translation mode. ONLCR is specified by Posix, and tends to be enabled by default (at least on Linux).

Reply:
Indeed, they fail without it. Thanks.